docs/src/python/sphinx.rst

```diff
@@ -48,7 +48,8 @@ The ``structure`` mode will show the structures only
 and the ``map`` mode only the property map
 
 .. chemiscope:: ../datasets/showcase.json.gz
-    :mode: map 
+    :mode: map
+    :warning_timeout: 0.0 
```

Comment:
should we make this the default in sphinx?

Reply:
After thinking a bit I made the default 2s, because if one plays around with the viewer then it may trigger warnings we want to show